test_suite/stdlib.jsonnet

```diff
@@ -203,20 +203,25 @@ std.assertEqual(std.endsWith('food', 'food'), true) &&
 std.assertEqual(std.endsWith('food', 'omgfood'), false) &&
 std.assertEqual(std.endsWith('food', 'wat'), false) &&
 
+local veryLongString = std.join('', std.repeat(['e'], 510));
```

Comment:
This is too slow for the regular tests. Please add it to `benchmarks/` instead (as a separate file). 

(BTW this really shouldn't be so damn slow. I wonder how go-jsonnet fares here).

Reply:
I moved the test to benchmars.